src/analyzer.rs

```diff
@@ -318,14 +336,16 @@ mod tests {
     kind: Redefinition { first_type: "alias", second_type: "alias", name: "foo", first: 0 },
   }
 
+  const UNKNOWN_ALIAS_TARGET: &str = "alias foo := bar\n";
```

Comment:
I don't like these kinds of constants which do a very limited amount of deduplication. I'd just inline this in both places.